app/core/Tracker2/SegmentTracker.js

```diff
@@ -0,0 +1,166 @@
+import BaseTracker from './BaseTracker'
+
+// Copied from Segment analytics-js getting started guide at:
+// https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/quickstart/
+export function loadSegment () {
```

Comment:
Just clarifying that this isn't loaded yet. It looks like this is still commented out in `Tracker2`.
I like this architecture of having the various trackers split up with a tracker that combines them!

Reply:
Yup thats right, this is a placeholder that will be integrated in the near future.